### PR TITLE
Create issue-template.md

### PR DESCRIPTION
**Pull request by lukehansell**
#### What does this PR do? (please provide any background)

In order to improve our transparency/history/issue noting we want to capture specific information in Github issues for app-platform. This document outlines the process and questions that should be answered when raising issues for app-platform.
- [ ] +1
- [ ] +1 and :pineapple:

### CODE REVIEW

**Comment by hx-markterry** [comment]
I think nows the time to think about why we have two issue tracking systems.

**Comment by shackpank** [comment]
I'm coming round to that as well.

For public repos like this one, and some of the react components and helpers we've got, the template has odd specifics about the process we have for working on the private ones.

**Comment by lukehansell** [comment]
@hx-markterry I'm open to discussion if there is an issue with how app-platform currently reports bugs?
@shackpank any specific feedback on points could I address to make the template more diverse?

**Comment by hx-markterry** [comment]
@lukehansell i've added it a point to our group tech meeting, it was scheduled for this week, but it was postponed due to many absenses

**Comment by lukehansell** [comment]
@hx-markterry @shackpank has there been any movement on this from the group tech meeting? I'd like to get this (or some version of it) merged so that we can start pushing standards in our issue creation

**Comment by hx-markterry** [comment]
@hpoom has taken this as one of his actions to resolve the issue with pod's using different tracking systems.

File: issue-template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+If you know the work will be undertaken by a specific POD then add a label with the POD name to the issue.
+
+If you are creating multiple issues around a topic or with a common goal then create a milestone and add all relevant issues to it.
+
+Please, when you begin working on an issue assign yourself to it and treat the issue as a mind-dump. Write all problems/ideas/blockers/designs/failures etc. that you have on there so that people can easily follow decisions you have made and discussion you have had. Try to update your issues at least once a day.
+
+```
+#What is the problem you're trying to solve?
+
+#What is the benefit in solving it?
```
Comment (jamesmcneil) [inline issue-template.md:17]: I would suggest using `customer benefit` here as we have a high focus on customer impact of our work?

File: issue-template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+#What is the expected result of this issue?
+
+#What unknowns are there to deal with?
```
Comment (jamesmcneil) [inline issue-template.md:21]: Would it be simpler to ask about impact on other systems, as you cannot know the unknowns?

**Comment by shackpank** [comment]
Here I am, closing this issue. Too much internal jargon.

**Pull request closed without merging**